app/views.py

```diff
@@ -3,12 +3,18 @@
 import re
 from json import loads
 
-from app import app, models, sql_session, viewsModel
```

Comment:
I started refactoring views.py and as I've been organizing and rewriting it to make it REST compliant, views_models.py is kind of naturally getting sorted into functions by object.  I think once I'm done, it will be pretty straightforward to turn the functions into class methods :-)  I'll let you know how it goes!